src/sentry/testutils/fixtures.py

```diff
@@ -457,6 +458,8 @@ def create_event(self, event_id=None, **kwargs):
             kwargs['group'] = self.group
         kwargs.setdefault('project', kwargs['group'].project)
         kwargs.setdefault('data', copy.deepcopy(DEFAULT_EVENT_DATA))
+        kwargs.setdefault('platform', kwargs['data'].get('platform', 'python'))
+        kwargs.setdefault('message', kwargs['data'].get('message', 'message'))
```

Comment:
I was afraid of these changes but:

1. Uh, I thought they were required fields anyway?
2. No existing tests broke.